instrumentation/servlet/README.md

```diff
@@ -3,8 +3,9 @@
 ## Settings
 
 | System property                                                        | Type    | Default | Description                                         |
-| ---------------------------------------------------------------------- | ------- | ------- | --------------------------------------------------- |
+|------------------------------------------------------------------------| ------- | ------- |-----------------------------------------------------|
 | `otel.instrumentation.servlet.experimental-span-attributes`            | Boolean | `false` | Enable the capture of experimental span attributes. |
+| `otel.instrumentation.servlet.experimental.enduser-span-attributes`    | Boolean | `false` | Enable the capture of `enduser.id` span attribute.  |
```

Comment:
do you think it makes sense to generalize this from `.servlet.` to `.common.` in anticipation of #9400?

Reply:
I thought about that, but was unclear where the common flag could be documented (I guess it is in the website repo). So just added it for the servlet now. What do you suggest for the flag name? Is `otel.instrumentation.common.capture-enduser.enabled` that was suggested in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/9400 fine?

Reply:
> What do you suggest for the flag name? Is `otel.instrumentation.common.capture-enduser.enabled` that was suggested in #9400 fine?

👍